### DISCUSSION TITLE
AbfallNavi -need help to run the python Script for the necessary attributes

### DISCUSSION THREAD

**Discussion by Spartacus68** [mampfes/hacs_waste_collection_schedule#1628] [Q&A]
Hi,

I do not understand how to run the python Script for AbfallNavi in order to find the right service for my town. can someone please help me to the steps I have to do for this? and how can I install the inquire Module?

**Comment by 5ila5** [comment]
1. Install python if not already done
2. you need to download the python script as raw text ([right click -> save as] will probably download a html file but with the .py  file ending) https://raw.githubusercontent.com/mampfes/hacs_waste_collection_schedule/master/custom_components/waste_collection_schedule/waste_collection_schedule/wizard/abfallnavi_de.py
3. install inquire using `pip install inquire` in a command line window
4. run the script (python abfallnavi_de.py)

We plan to rely less on these scripts as not everyone find it easy to run them. I updated the documentation in #1635 to now show the service ids you should be able to configure this source without needing the wizzard.
